cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfParser.h

```diff
@@ -14,6 +14,8 @@ class FXmlNode;
 
 struct UrdfJointDesc;
 
+static const float unit_conversion_m_to_cm = 100.0f; // unit conversion from m to cm
```

Comment:
Don't need this here. Make it a `const float M_TO_CM = 100.0f` in the file where it is used.

Reply:
fixed as you suggested. It is used in multiple places now and not sure if we would use it in the future. We could change the design if it appears more frequently than we expect later.